packages/form/addon/components/cf-field/label.js

```diff
@@ -0,0 +1,22 @@
+import { getOwner } from "@ember/application";
+import Component from "@glimmer/component";
+
+/**
+ * Label component of the CfField
+ *
+ * @class CfFieldLabelComponent
+ */
+export default class CfFieldLabelComponent extends Component {
+  get config() {
+    return getOwner(this).resolveRegistration("config:environment");
+  }
+
+  get useMandatoryAsterisk() {
+    const { USE_MANDATORY_ASTERISK = false } =
+      this.config["ember-caluma"] || {};
+
+    return [true, false].includes(this.args?.useMandatoryAsterisk)
+      ? this.args?.useMandatoryAsterisk
+      : USE_MANDATORY_ASTERISK;
```

Comment:
```suggestion
    return this.args.useMandatoryAsterisk ?? USE_MANDATORY_ASTERISK
```

Should almost do the same and would be much easier to read. Or did I miss anything (besides requiring a boolean value to be passed)?

Reply:
Would be more beautiful, I agree, but if someone mistakenly set it with "false" as string, it will be true - so i wanted to make sure, that this argument should be a true/false

Reply:
I totally see where this is coming from. However, I'd still drop this check as we don't do this anywhere else for the same use-case. I think the proper way to solve or prevent this problem would be using typescript & glint which we don't do at the moment.